packages/language-server/src/utils/files/resolveTemplate.ts

```diff
@@ -0,0 +1,41 @@
+import { fileStat } from './fileStat';
+
+// HACK: Maybe we should move these to a configuration setting?
+const extensions = [`.twig`, `.html`];
+
+const indexTemplateFilenames = [`index`];
+
+/**
+ * Searches for a template file, and returns the first match if there is one.
+ */
+export const resolveTemplate = async (
+    pathToTwig: string,
+): Promise<string | null> => {
+    let stats = await fileStat(pathToTwig);
+
+    if (stats && stats.isFile()) {
```

Comment:
nit: we can reuse the same pattern as below:

```suggestion
    if (stats?.isFile()) {
```